ui/components/ui/icon/README.mdx

```diff
@@ -31,7 +31,7 @@ The following describes the props and example usage for this component.
 Use the `size` prop to change the size of the icon it should adhere to an `8px` grid e.g. `16px`, `24px`, `32px`
 
 <Canvas>
-  <Story id="components-ui-icon--size" />
+  <Story id="components-ui-icon--size-story" />
```

Comment:
Doesn't look like the Size story is updated in the `icon.stories.js` folder. This will be deprecated and removed soon but can't hurt

Reply:
So it is! My bad 

<img src="https://media1.giphy.com/media/jUwpNzg9IcyrK/giphy.gif"/>